score-client/src/main/resources/application.yml

```diff
@@ -29,6 +29,9 @@ client:
   accessToken: "${accessToken:}" # Alias for backwards compatibility
   connectTimeoutSeconds: 60
   readTimeoutSeconds: 60
+  # path of dir with WRITE access for score client upload state files
+  # if empty uses parent dir of current file to upload
+  uploadStateParentDirPath: ""
```

Comment:
`uploadStateDir`

Just don't think we need to be that verbose. The directory that contains the state by definition is the parent directory of the state.